How-To-Host-Feed.md

```diff
@@ -1,7 +1,7 @@
 # How To Host Your Own [Private/Internal/Public] Package Repository Server (aka Package Feed)
 
 ## Why?
-Please read [[Depending on the chocolatey.org as an organization|ChocolateyFAQs#should-my-organization-depend-on-use-the-community-feed-httpschocolateyorgpackages]]
+Please read [[Depending on the chocolatey.org as an organization|ChocolateyFAQs#should-my-organization-depend-on-use-the-community-feed-httpschocolatey.orgpackages]]
```

Comment:
Think we need to fix it so the link is the same for both docs on website and wiki. The wiki likes the former, the docs links generation uses the ".".